### ISSUE TITLE
icc not configured

### ISSUE THREAD

**Issue by chises** [fmbiete/Z-Push-contrib#208]
Hi all,
i am using BackendCombined
IMAP+CALDAV+CARDAV.

if i delete / create a IMAP Folder i get These line in my errorlog.
16/06/2015 10:22:02 [ 1406] [ERROR] [test@domain.de] [ac571db412e642529ad01b7dd7d72a31] ImportChangesCombined->Config() icc not configured

Here is my imap Folder test Output.
root@debian:/var/www/z-push/sql# php ../testing/testing-imap_folder_list.php
Evaluating Trash
Evaluating Spam
Evaluating Sent Items
Evaluating INBOX
Evaluating Drafts

New Folders dont show up on my IMAP Folders in webinterface/php script.
Suggestions?

**Comment by fmbiete** [comment]
Yes, I believe that functionality is not complete at the moment.

**Comment by chises** [comment]
thank you @fmbiete 

other questions: email sent/receive is fine. if i create imap Folder via IMAP Connections everything is fine. But if i try to create These Folder with ActiveSync, the Folder is not created..i can not see any issues.

using Debian +dovecot:

// Since I know you won't configure this, I will raise an error unless you do.
// When configured set this to true to remove the error
define('IMAP_FOLDER_CONFIGURED', true);

// Folder prefix is the common part in your names (3, 4)
define('IMAP_FOLDER_PREFIX', '');

// Inbox will have the preffix preppend (3 & 4 to true)
define('IMAP_FOLDER_PREFIX_IN_INBOX', false);

// Inbox folder name (case doesn't matter) - (empty in 4)
define('IMAP_FOLDER_INBOX', 'inbox');

// Sent folder name (case doesn't matter)
define('IMAP_FOLDER_SENT', 'sent');

// Draft folder name (case doesn't matter)
define('IMAP_FOLDER_DRAFT', 'drafts');

// Trash folder name (case doesn't matter)
define('IMAP_FOLDER_TRASH', 'trash');

// Spam folder name (case doesn't matter). Only showed as special by iOS devices
define('IMAP_FOLDER_SPAM', 'SPAM');

// Archive folder name (case doesn't matter). Only showed as special by iOS devices
define('IMAP_FOLDER_ARCHIVE', 'ARCHIVE');

**Comment by tickerguy** [comment]
This by the way becomes a HUGE problem if you set up a new account and there is no "Trash" folder on the server; when you go to delete a message it gets HARD deleted instead because the Trash folder doesn't exist.....

**Comment by chises** [comment]
ah nice, thanks. now i know why i cant create Folders.. :)
can someone say if/ when this will be implemented. acutally i am working on my exam, so i can not ivestigate any time. i am sorry :-(
do we have to wait for official z-push to implement this?

**Comment by tickerguy** [comment]
I may hack on this at some point..... I tend to focus a bit of energy on things when they annoy me sufficiently :-)

**Comment by FoZo** [comment]
Hello,
are there any plans to implement this?
